using-mobile-extensions/places-extension-1/places-extension/places-api-reference.md

```diff
@@ -5,13 +5,53 @@
 When a device crosses one of your app's pre-defined Places region boundaries, the region and event type are passed to the SDK for processing.
 
 {% tabs %}
+
 {% tab title="Android" %}
-### ProcessGeofenceEvent
+
+### ProcessGeofence
+
+Process a single `Geofence` region event for the provided `transitionType`.
```

Comment:
please remove "single"...you can leave it as "Process a..." or change it to "Process one..."